src/reference/asciidoc/http.adoc

```diff
@@ -58,6 +58,26 @@ If it does in fact locate that bean, then the support for MultipartFiles will be
 Otherwise, it will fail when trying to map a multipart-file request to a Spring Integration Message.
 For more on Spring's support for MultipartResolvers, refer to the http://static.springsource.org/spring/docs/current/spring-framework-reference/html/mvc.html#mvc-multipart[Spring Reference Manual].
 
+[NOTE]
+====
+If you wish to proxy a `multipart/form-data` to another server, it may be better to keep it in raw form.
+To configure for this situation, do not configure the `multipartResolver` bean, configure the endpoint to expect
```

Comment:
Actually, there is a work-around - you can inject a multipart resolver directly into the endpoint.

It's not supported by the namespace, but it means we can have a mixture of endpoints that do/do not use a resolver.

See `HttpRequestHandlingEndpointSupport.prepareRequest` - if the `DispatcherServlet` has not already decoded the multipart, and the endpoint has a resolver, it will be used to decode.